lib/features.js

```diff
@@ -27,6 +28,18 @@ module.exports = config => {
     },
 
     /**
+     * Return whether or not push notification for sign in confirmation is
+     * enabled for a given user, based on their uid and email address.
+     *
+     * @returns {boolean}
+     */
+     isPushLoginVerifyEnabled(uid, email) {
+      return pushNotificationForSignIn && pushNotificationForSignIn.enabled &&
+        isSampledUser(pushNotificationForSignIn.sampleRate, uid, 'pushNotificationForSignIn' ||
```

Comment:
I think this has a parenthesis in the wrong place, should be `isSampledUser(pushNotificationForSignIn.sampleRate, uid, 'pushNotificationForSignIn')`